javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -36,8 +36,23 @@ When creating a new repo on GitHub, there is an option to specify a `.gitignore`
 1. Next, set up your restaurant site to use tabbed browsing to access the Menu and Contact pages. Look at the behavior of this [student's live preview site](https://web.archive.org/web/20221024060550/https://eckben.github.io/bearysBreakfastBar/) for visual inspiration.
     1. Put the contents of each "tab" inside of its own module. Each module will export a function that creates a div element, adds the appropriate content and styles to that element and then appends it to the DOM.
     1. Write the tab-switching logic inside of `index.js`. You should have event listeners for each button in the header navbar that wipes out the current contents of `div#content` and then runs the correct 'tab module' to populate it with the new contents again.
-1. Let's deploy to GitHub pages! First, we will need to bundle our application into `dist` by running `npx webpack`. Unfortunately, we need to do a little more work to deploy to GitHub pages, because GitHub Pages tries to look for an `index.html` *in the root of your project*, but yours is inside `dist`! We will need to do a few steps to push *the contents* of your `dist` directory to its own branch on GitHub, which will then have a root-level `index.html` for GitHub pages to serve.
-    1. Follow the instructions in this [gist about deploying your dist subdirectory to GitHub pages](https://gist.github.com/cobyism/4730490). EZPZ!
-    1. Recall that the **source branch** for GitHub Pages is set in your repository's settings. Get this changed to the `gh-pages` branch.
+
+#### Deployment
+
+Let's deploy your project to GitHub pages! This is a little more work than it has been for previous projects, because GitHub Pages tries to look for an `index.html` *in the root of your project*, but yours is inside `dist`! We will need to do a few steps to push *the contents* of your `dist` directory to its own branch on GitHub, which will then have a root-level `index.html` for GitHub pages to serve. You don't need to know exactly what all the commands do - as long as you follow the instructions below carefully you should be fine. You can use these instructions to deploy your project initially, and also redeploy it again if you make more changes to your project later.
+
+1. Make a new branch to deploy from by running `git branch gh-pages`. You only need to do this the first time you deploy. The rest of the steps should be done every time you deploy or redeploy your project.
+1. Make sure you have all your work committed. You can use `git status` to see if there's anything that needs committing.
+1. Run `git checkout gh-pages && git merge main --no-edit` to change branch so that you're ready to deploy.
+1. Now let's bundle our application into `dist` with your build command. For now, that's `npx webpack`.
+1. Now there are a few more commands. Run each of these in order.
```

Comment:
Thoughts?

```suggestion
1. Now there are a few more commands that will unpack `dist` and push its contents to your `gh-pages` branch on GitHub. Run each of these in order:
```

Reply:
I'm not sure on this one. The colon is definitely valid, but I think "unpack dist and push" has already been explained, and any more detail would be getting into the weeds too much

Reply:
Fair point. The colon shall go in though :+1: